docs/docs/code/python/using-data-stores/README.md

```diff
@@ -121,6 +135,36 @@ What's the difference between `data_store["key"]` and `data_store.get("key")`?
 * `data_store.get("key", "default_value")` will return `"default_value"` if the key doesn't exist on the Data Store.
```

Comment:
Syntax highlighting is off on this line and above, something in the code / Markdown looks like it's misformatted